app/uk/gov/hmrc/plasticpackagingtax/registration/views/partner/partner_check_answers_page.scala.html

```diff
@@ -34,6 +34,15 @@
 
 @(partner: Partner)(implicit request: JourneyRequest[_], messages: Messages)
 
+@changePartnerNameLink = @{
+    if(partner.canEditName) {
+        Some(partnerRoutes.PartnerNameController.displayExistingPartner(partner.id))
+    } else {
+        None
+    }
+}
```

Comment:
Nice but this is now slightly inconsistent with the amendment of the top-level partnership which does not permit the name to be changed without making a change to the organisation type. We might want to create a ticket to have that changed to operate like you've done things here?